.github/actions/update-mod-version/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Update module version'
+description: 'Update plugin module version in target repo.'
+inputs:
+  branch:  # id of input
+    description: 'Branch of target repo to checkout.'
+    required: true
+    default: 'master'
+  update-strategy:  # id of input
+    description: 'How to update the go module.'
+    required: true
+    default: '@latest'
```

Comment:
IMHO I would suggest omitting the `@` here, as it's really part of the `go get` command rather than the update strategy, and the `@` makes the meaning of the syntax look a little confusing to me.